docs/introduction/tutorials_intro.md

```diff
@@ -110,145 +114,158 @@ And the following tutorial:
+| Network status | [Network status snippet](../apis/network-status.html#example) | | |
+| Wi-Fi | [Wi-Fi snippet](../apis/wi-fi.html#wi-fi-example) | | [Wi-Fi official example](https://github.com/ARMmbed/mbed-os-example-wifi) |
 
 ## Secure socket APIs
 
-| API | Examples | Tutorials |
-| --- | --- | --- |
-| DTLSSocket | [DTLSSocket example](../apis/secure-socket-apis.html#dtlssocket-example) | |
-| TLSSocket | [TLSSocket example](../apis/tlssocket.html#tlssocket-example) | [Secure communication tutorial](../apis/tls-tutorial.html) |
+| API | Snippets | Tutorials | Official examples |
+| --- | --- | --- | --- |
+| DTLSSocket | [DTLSSocket snippet](../apis/secure-socket-apis.html#dtlssocket-example) | | |
+| TLSSocket | [TLSSocket snippet](../apis/tlssocket.html#tlssocket-example) | [Secure communication tutorial](../apis/tls-tutorial.html) | [TLS socket official example](https://github.com/ARMmbed/mbed-os-example-tls-socket) |
 
 ## DNS API
 
-| API | Examples | Tutorials |
+| API | Snippets | Tutorials |
 | --- | --- | --- |
-| DNS Resolver | [DNS Resolver example](../apis/dns-apis.html#dns-resolver-example) | |
+| DNS Resolver | [DNS Resolver snippet](../apis/dns-apis.html#dns-resolver-example) | |
 
 ## Bluetooth (BLE) APIs
 
-We have placed all of our BLE examples in a single [GitHub repository](https://github.com/ARMmbed/mbed-os-example-ble).
+We have placed all of our official BLE examples in a single [GitHub repository](https://github.com/ARMmbed/mbed-os-example-ble).
 
 ## NFC APIs
 
-| API | Examples |
+| API | Snippets |
 | --- | --- |
-| MessageBuilder | [MessageBuilder example](../apis/nfc-apis.html#messagebuilder-example) |
-| MessageParser | [MessageParser example](../apis/messageparser.html#messageparser-example) || Mesh | | [Mesh tutorial](../apis/connectivity-tutorials.html) <br>  [Light control tutorial](../apis/light-control.html) |
-| NFCController | [NFCController example](../apis/nfccontroller.html#nfccontroller-example) |
-| NFCEEPROM | [NFCEEPROM example](../apis/nfc-eeprom.html#nfceeprom-example) |
-| SimpleMessageParser | [SimpleMessageParser example](../apis/simplemessageparser.html#simplemessageparser-example) |
+| MessageBuilder | [MessageBuilder snippet](../apis/nfc-apis.html#messagebuilder-example) |
+| MessageParser | [MessageParser snippet](../apis/messageparser.html#messageparser-example) || Mesh | | [Mesh tutorial](../apis/connectivity-tutorials.html) <br>  [Light control tutorial](../apis/light-control.html) |
+| NFCController | [NFCController snippet](../apis/nfccontroller.html#nfccontroller-example) |
+| NFCEEPROM | [NFCEEPROM snippet](../apis/nfc-eeprom.html#nfceeprom-example) |
+| SimpleMessageParser | [SimpleMessageParser snippet](../apis/simplemessageparser.html#simplemessageparser-example) |
 
+And the [official examples repository](https://github.com/ARMmbed/mbed-os-example-nfc).
```

Comment:
The two NFC examples should be linked in the tables above. We have
- https://github.com/ARMmbed/mbed-os-example-nfc/blob/master/NFC_SmartPoster/README.md
- https://github.com/ARMmbed/mbed-os-example-nfc/tree/master/NFC_EEPROM/README.md